docs/model-dev-guide/api-guides/batch-processing/batch-process-api-ug.rst

```diff
@@ -151,7 +162,37 @@ example.
 
            self.output = []
 
-Step 2: Initialize the Dataset
+(Optional) Step 2: Link the Run to a Checkpoint or Model Version
+================================================================
```

Comment:
Step 2: Link the Run to a Checkpoint or Model Version (Optional) 
================================================================